### PR TITLE
entity matching pipelines, limits, 0.23

### PR DESCRIPTION
**Pull request by sanderland**

### CODE REVIEW

File: cognite/experimental/data_classes/contextualization.py [hunk 1]
```diff
@@ -218,3 +220,79 @@ def description(self):
...
+        status_timestamp=None,
+        cognite_client=None,
+    ):
+        """
+        external_id, name, description: standard fields for a resource.
+        model_id: id of the entity matching model to deploy
+        match_from, match_to: descriptions
+        matches: true matches to use in training
+        rules: list of matching rules
+        The other fields should be left empty when creating a pipeline, and return status information on successful creation and retrieval.
```
Comment (HMEiding) [inline cognite/experimental/data_classes/contextualization.py:253]: Should the other properties have descriptions as well?

File: cognite/experimental/_api/entity_matching.py [hunk 2]
```diff
@@ -8,13 +8,118 @@
...
+class EntityMatchingPipelineRunsAPI(ContextAPI):
+    _RESOURCE_PATH = EntityMatchingPipeline._RESOURCE_PATH + "/run"
+    _LIST_CLASS = EntityMatchingPipelineRunList
+
+    def retrieve(self, id: int) -> EntityMatchingPipelineRun:
+        """Run pipeline
+
+        Args:
+            id: id of the pipeline to run.
+            external_id: external id of the pipeline to run.
```
Comment (HMEiding) [inline cognite/experimental/_api/entity_matching.py:27]: The `external_id` is not an argument...

**Review by HMEiding — APPROVED** [review]
A couple of comments about the docstrings, but otherwise LGTM.

File: cognite/experimental/_api/entity_matching.py (hunk 2 above)
Comment (sanderland) [reply cognite/experimental/_api/entity_matching.py:27]: fixed.

File: cognite/experimental/data_classes/contextualization.py (hunk 1 above)
Comment (sanderland) [reply cognite/experimental/data_classes/contextualization.py:253]: we're a bit under-documented in these data classes, but they're obvious enough for now

**Pull request merged**